public/docs/ts/latest/guide/template-syntax.jade

```diff
@@ -553,21 +554,21 @@ block dart-type-exception-example
 a(id="one-time-initialization")
 :marked
   ### One-time string initialization
-  We *should* omit the brackets when all of the following are true:
+  The brackets *should* be omited when the following conditions are true:
```

Comment:
Avoid passive voice! (I think it'd be "omitted" if we let it in, but we won't. :) Also, it's important to say *all*, for clarity. (Otherwise, someone might think it's *any*.) Instead, maybe:

Omit the brackets when all of the following conditions are true: